### PR TITLE
Widgets/Code-Highlight: adding dynamic-tags

### PR DESCRIPTION
**Pull request by nisan250**
ED-2242

### CODE REVIEW

File: assets/dev/scss/editor/panel/_controls.scss [hunk 1]
```diff
@@ -504,6 +504,8 @@ $devices: 'desktop', 'tablet', 'mobile';
 
 @import "controls/button";
 
+@import "controls/code";
+
 @import "controls/choose";
```
Comment (StyleShit) [inline assets/dev/scss/editor/panel/_controls.scss:509]: Order alphabetically.
```suggestion
@import "controls/choose";

@import "controls/code";
```

File: assets/dev/scss/editor/panel/controls/_code.scss [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+	&.elementor-control-dynamic-value {
+
+		.elementor-control-dynamic-switcher {
+			border-radius: getValueByDirection(0 3px 3px 0, 3px 0 0 3px);
+			@include border-start-property(width, 0);
+		}
+	}
+
+	.elementor-control-dynamic-switcher {
+		border-radius: 0px 3px;
```
Comment (StyleShit) [inline assets/dev/scss/editor/panel/controls/_code.scss:12]: Why you don't use `getValueByDirection` for this?

**Review by StyleShit — CHANGES_REQUESTED** [review]

File: assets/dev/scss/editor/panel/controls/_code.scss (hunk 2 above)
Comment (matipojo) [reply assets/dev/scss/editor/panel/controls/_code.scss:12]: It's like the same CSS on the textarea.

File: assets/dev/scss/editor/panel/controls/_code.scss [hunk 3]
```diff
@@ -0,0 +1,39 @@
+.elementor-control-type-code {
+
+	&.elementor-control-dynamic-value {
```
Comment (matipojo) [inline assets/dev/scss/editor/panel/controls/_code.scss:3]: Try to share the code with the textarea control.

**Review by matipojo — CHANGES_REQUESTED** [review]

File: assets/dev/scss/editor/panel/_controls.scss (hunk 1 above)
Comment (nisan250) [reply assets/dev/scss/editor/panel/_controls.scss:509]: Done.

File: assets/dev/scss/editor/panel/controls/_code.scss (hunk 3 above)
Comment (nisan250) [reply assets/dev/scss/editor/panel/controls/_code.scss]: Done.

**Review by matipojo — APPROVED** [review]

**Review by kobizz — APPROVED** [review]

**Pull request merged**